middleware.js

```diff
@@ -1,4 +1,44 @@
 var http = require('http');
 
+var app = {};
+
+app.handle = function (req, res) {
+  var index = 0;
+  function next() {
+    if(index >= app.middlewares.length) {
+      return;
+    }
+    var middleware = app.middlewares[index++];
+    if(middleware.path === '/' || middleware.path === req.url) {
+      middleware.handle(req, res, next);
+      return;
```

Comment:
@yorkie Could you add a test case for it? This should be a design problem (as all the other Unit test problems)